docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@loopback/docs",
-  "version": "0.16.4",
+  "version": "0.16.5",
```

Comment:
We should not update `version`. The release process will automatically update it.

Reply:
👍  I was waiting for this comment.